content/questions/foundations/question-103.md

```diff
@@ -0,0 +1,15 @@
+---
+archetype: "questions"
+title: "Question 103"
+question: "Which of the following Git commands allow you to create a new branch and start working on it in one line? (Select two.)"
+draft: false
+---
+
+
+> https://git-scm.com/docs/git-switch
+1. [x] git checkout -b `<new_branch_name>`
+1. [ ] git branch -c `<new_branch_name>`
+1. [x] git switch -c `<new_branch_name>`
+1. [ ] git merge -m `<new_branch_name>`
+1. [ ] git switch -m `<new_branch_name>`
+1. [ ] git checkout -m `<new_branch_name>`
```

Comment:
This is the format for single choice questions.  For multi choice questions it's required to use 
` - [x]` 
instead of
`1. [x]`. 


```suggestion
- [x] git checkout -b `<new_branch_name>`
- [ ] git branch -c `<new_branch_name>`
- [x] git switch -c `<new_branch_name>`
- [ ] git merge -m `<new_branch_name>`
- [ ] git switch -m `<new_branch_name>`
- [ ] git checkout -m `<new_branch_name>`
```

For future reference -> If you create the question files using Hugo commands as described in [CONTRIBUTING](https://github.com/FidelusAleksander/ghcertified/blob/master/CONTRIBUTING.md#adding-new-question) they fill up with ready to go [template](https://github.com/FidelusAleksander/ghcertified/blob/master/archetypes/questions.md?plain=1#L19)